.github/buildomat/jobs/build-and-test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#:
+#: name = "build-and-test"
+#: variety = "basic"
+#: target = "helios"
```

Comment:
Just checking -  does the [OS target](https://doc.rust-lang.org/reference/conditional-compilation.html#target_os) here show up as "illumos" or "helios"?

Reply:
(That is, from the perspective of a conditionally compiled piece of rust code)

Reply:
The Rust-visible OS string is definitely `illumos`, regardless of which distribution you end up on.

In this context, "target" is a buildomat-specific term that is supposed to denote the kind of worker you want the job to run it.  The only supported target at this minute is `helios`, and I'm also now thinking maybe I had better pick a less over-loaded word haha.